urdf_model_state/include/urdf_model_state/model_state.h

```diff
@@ -49,6 +49,15 @@
 
 namespace urdf{
 
+//round is not defined in C++98
+//So in Visual Studio is necessary to define it
+#ifdef _MSC_VER
```

Comment:
I remember that I was developing this PR to compile a project depending on urdfdom in VS2012, so I think this is make sense.